runtime/autoload/netrw.vim

```diff
@@ -11454,24 +11454,29 @@ fun! s:NetrwLocalRmFile(path,fname,all)
 
   else
    " attempt to remove directory
+   let rmfile= substitute(rmfile,'[\/]$','','e')
    if !all
     echohl Statement
     call inputsave()
-    let ok= input("Confirm *recursive* deletion of directory<".rmfile."> ","[{y(es)},n(o),a(ll),q(uit)] ")
+    " make it be a bit harder to make mistake by inputting upper case
+    let ok= input("Confirm *recursive* deletion of directory<".rmfile."> ","[{Y(ES)},n(o),A(LL),q(uit)] ")
```

Comment:
Now you are unexpectedly changing the key-bindings. This will break peoples habits. I don't think this is appreciated by long-time users. (besides, it actually looks funny, that some keys need to be upper-cased and others not).

Reply:
well, i am not against just back out the https://github.com/vim/vim/pull/14742, then kept the appreciated long-time users' habits, or you had better idea not to so easy or high possibility to make mistake to rm all?

Reply:
What mistake, there is still a confirmation dialog

Reply:
1. lower case is easy to mis-type, like a 'hit-enter' happened.
2. it was designed to: if 'all=1' then no 'confirm', else req a confirm, and confirm to only delete('rm', 'd') not 'rf'.